frontend/views/containers/chatroom/ChatMembers.vue

```diff
@@ -12,7 +12,7 @@
 
   ul.c-group-list
     list-item(
-      v-for='{username, displayName, picture } in directMessages'
+      v-for='{username, displayName, picture} in priateDirectMessages'
```

Comment:
`priateDirectMessages` is misspelled, should be `privateDirectMessages` I think?